source/_integrations/youless.markdown

```diff
@@ -11,6 +11,7 @@ ha_codeowners:
   - '@gjong'
 ha_platforms:
   - sensor
+  - energy
```

Comment:
`energy` is not a platform.

Reply:
F...
I somehow saw this as being under `ha_category`....
https://github.com/home-assistant/home-assistant.io/pull/19343